static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -53,16 +53,26 @@ export function useFeedbackOnboardingDrawer() {
 
   useEffect(() => {
     if (isActive && hasProjectAccess) {
-      openDrawer(() => <SidebarContent />, {
+      openDrawer(() => <DrawerContent />, {
         ariaLabel: t('Getting Started with User Feedback'),
-        onClose: () => {
-          SidebarPanelStore.hidePanel();
-        },
+        // Prevent the drawer from closing when the query params change
+        shouldCloseOnLocationChange: location =>
```

Comment:
Realized that we needed to add some options to the feedback one to make it work correctly. 

This is very similar to the replay onboarding one, so I think there is an opportunity for a common hook or component. Don't want to make that abstraction too early though, going to wait until after I convert more of these